README.md

```diff
@@ -1,21 +1,58 @@
-# 🇨🇦 🇬🇹 🇰🇷 🇱🇧 Olympick app 🏴󠁧󠁢󠁷󠁬󠁳󠁿 🇵🇰 🇮🇷 🇬🇧 🇨🇴 
 
-Currently there is no way to create a viewing schedule for the Olympics that only includes specific sports.
+# 🇨🇦 🇬🇹 🇰🇷 🇱🇧 🇯🇵 Olympick 🏴󠁧󠁢󠁷󠁬󠁳󠁿 🇵🇰 🇮🇷 🇬🇧 🇨🇴 
 
-This app allow a user to : <br/>
- <li>input their favourite Olympic sport's name and get all the schedules of it.</li><br/>
- <li>input their favourite Olympic sport's name  and get the schedules of specific event of it.</li><br/>
+Currently there is no way to create a viewing schedule for the Olympics that only include specific sports. 
 
-# Tools
+Olympick aims to address this! 
 
-<li> 2020 Olympic Public API : (https://olypi.com/) </li>
-<li> SQL databases</li>
-<li> Python language</li>
-<li> Python Libraries : <strong>Flask</strong>,<strong>Collections</strong>,etc </li>
-<li>Tests : Unit tests and mock tests</li>
+This app allows a user to : <br>
+ <li>Register or log in.</li><br>
+ <li>Input their favourite sports and receive a schedule of all the events happening for those sports.</li><br/>
+ <li>Save chosen events to their personalised schedule.</li>
```

Comment:
instead of save chosen events we can say save their favourite events for specific sports in the UK time and they can remove those events they do not want from their schedule